media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-module
+
+DESCRIPTION="Set of utilities to manage TV listings stored in the XMLTV format"
+HOMEPAGE="http://xmltv.org"
```

Comment:
`https://` also please include the Github page here.

Reply:
Added Github page.
Found more accurate homepage. Homepage is only available at `http://`... I did not find a secure version.